src/Microsoft.OData.Edm/Csdl/Serialization/EdmModelCsdlSchemaXmlWriter.cs

```diff
@@ -24,12 +25,17 @@ internal class EdmModelCsdlSchemaXmlWriter : EdmModelCsdlSchemaWriter
     {
         protected XmlWriter xmlWriter;
         protected readonly string edmxNamespace;
+        private readonly CsdlXmlWriterSettings writerSettings;

```

Comment:
I'd prefer consistent naming across class members and parameters and across different files. Currently, this is named `writerSettings`, `settings`, `csdlXmlWriterSettings` across different files. Consistent naming makes it easier to relate things, IMO.